### PR TITLE
Tags glossary edit

### PR DESCRIPTION
**Pull request by jdmooney**

### CODE REVIEW

File: userguide/subscription/includes/glossary.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+*Plan* - A Catalog object that defines how a Product is to be paid for. Plans can have multiple phases (e.g. trial, discount, evergreen). The phase specifies the frequency of payment and the prices in multiple currencies. Each phase can have fixed and recurring prices associated with it. Each Product can have multiple Plans specifying different ways that the Product can be purchased (e.g. a monthly Plan and an annual Plan).
 
-*Price list* - Price lists are collections of plans that might be associated with a particular program (e.g. an affiliate program or a discount program). A plan can be in many price lists. There must be a default price list that is used when no pricelist is specified. When a subscription changes plans, Catalog rules can be used to determine which Price list the new Plan should use.
+*Price list* - A collection of Plans that might be associated with a particular program (e.g. an affiliate program or a discount program). A Plan can be in many Price Lists. There must be a default Price List that is used when no Price List is specified. When a Subscription changes Plans, Catalog Rules can be used to determine which Price List the new Plan should use.
```
Comment (sbrossie) [inline userguide/subscription/includes/glossary.adoc:11]: > A Plan can be in many Price Lists.

I realize the doc was already saying that, but I am wondering if this is really true. Would you be able to make an experiment and try it out?

Comment (jdmooney) [reply userguide/subscription/includes/glossary.adoc:11]: I'm not sure, but I will try to check on this.

Comment (jdmooney) [reply userguide/subscription/includes/glossary.adoc:11]: I have thought awhile about this. It seems reasonable enough that one price list could contain some of the same plans as another one. I created a test catalog that included this case and it passed the validation tool. I don't know any other way to check it.

Comment (sbrossie) [reply userguide/subscription/includes/glossary.adoc:11]: OK, good enough :-)

**Pull request merged**